### ISSUE TITLE
Bar Chart Examples

### ISSUE THREAD

**Issue by techniq** [techniq/layerchart#21]
- https://observablehq.com/@d3/zoomable-bar-chart
- https://observablehq.com/@d3/bar-chart-transitions
- https://observablehq.com/@d3/hierarchical-bar-chart

**Comment by risalfajar** [comment]
Could you also add an example of Grouped Bar Chart with Tooltip?

**Comment by techniq** [comment]
@risalfajar Yeah, this one is not as straight forward as it should be.  Typically I will attach the "item group data" to each item as only one is currently selected with the current `tooltip={{ mode: 'band' }}` heuristics.

I'll likely either update [createStackData](https://github.com/techniq/layerchart/blob/main/packages/layerchart/package/utils/stack.js#L4) to do automatically, or try to improve [TooltipContext](https://github.com/techniq/layerchart/blob/main/packages/layerchart/src/lib/components/TooltipContext.svelte#L146) `showTooltip()` mode handling to better handle this

**Comment by techniq** [comment]
@risalfajar Sorry for such a long delay, but finally getting around to tackling this as part of the [Simplified Charts PR](#224)

https://github.com/user-attachments/assets/ec73ec57-4c01-4e93-9e99-679dec315162

Still some work to go, but the new simplified `<BarChart>` will support group and/or stacking with tooltips, as well as rolling your own via `<Chart>` (which this video is using)
